server/writer.go

```diff
@@ -0,0 +1,42 @@
+package server
+
+import (
+	"time"
+
+	"github.com/relex/fluentlib/server/receivers"
+	"github.com/relex/gotils/logger"
+)
+
+func launchWriter(wlogger logger.Logger, receiver receivers.Receiver) chan<- receivers.ClientMessage {
+	outputChan := make(chan receivers.ClientMessage, 1000)
```

Comment:
how does this work? is this an endless for loop below?

Reply:
It ends when `outputChan` is closed, at https://github.com/relex/fluentlib/pull/6/files/4c97b33b61eaa5819822327722ced02925977af8#diff-e6176062de0db7f6044b9230ad49e69ed89f1398c0c04af775d42bb0f7721e19R21